README.md

```diff
@@ -146,6 +146,51 @@ title,content
 My Title,Hello World!
 ```
 
+### Creating a custom Task
```

Comment:
Can you add this section in the [Table of Contents](https://github.com/Shopify/maintenance_tasks#table-of-contents) too?